src/main/java/org/openstreetmap/atlas/utilities/testing/TestAtlasHandler.java

```diff
@@ -143,6 +163,12 @@ else if (fullySplit.size() == 1)
                 throw new CoreException("{} isn't a valid tag description", tagAndValue);
             }
         }
+        // Add a country code if one does not already exist
+        if (!iso.equals(ISOCountryTag.UNKNOWN_ISO_COUNTRY)
```

Comment:
What do you think about passing around an `Optional<String>` for the ISO code? Then Optional.empty can be the null value which feels a little cleaner than ISOCountryTag.UNKNOWN_ISO_COUNTRY